test/system/001-basic.bats

```diff
@@ -98,8 +98,13 @@ function setup() {
         skip "only applicable on a local run"
     fi
 
-    CONTAINER_HOST=foobar run_podman --log-level=info --help
-    is "$output" ".*defaulting to '--remote=true'" "CONTAINER_HOST sets --remote true"
+    CONTAINER_HOST=foobar run_podman --help
+    # Should not have --remote flag
+    echo $output  |  grep -v -qw -- "--remote"
```

Comment:
```suggestion
    echo $output  |  grep -v -qw -- "--remote"
    if [ $? -ne 0 ]; then
        die "Should not have --remote flag"
    fi
```

Reply:
Sorry to have been so late. I was reviewing this right when it merged. This will not actually work as intended, but I will address it in a followup PR.